modal_proto/api.proto

```diff
@@ -1069,6 +1069,7 @@ message FunctionGetOutputsResponse {
   repeated int32 idxs = 3;
   repeated FunctionGetOutputsItem outputs = 4;
   string last_entry_id = 5;
+  bool is_expired = 6;
```

Comment:
Does this require server-side changes? Normally we would make such client-server updates in three steps:

- Merge new protos in the client
- Merge server side changes
- Merge client-side feature code

Reply:
got it! the server side changes are [here](https://github.com/modal-labs/modal/pull/13423), and i'll make a separate pr that only changes the protos.